src/components/EditProfile/EditCondition.js

```diff
@@ -0,0 +1,174 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { withStyles } from "@material-ui/core/styles";
+import FormControl from "@material-ui/core/FormControl";
+import FormLabel from "@material-ui/core/FormLabel";
+import MenuItem from "@material-ui/core/MenuItem";
+import Select from "@material-ui/core/Select";
+import painHistoryData from "../Registration/literals/painHistoryData.js";
+
+const styles = theme => ({
+  root: {
+    display: "flex",
+    flexFlow: "column wrap",
+    maxWidth: 250
+  },
+  formControl: {
+    margin: theme.spacing.unit * 3,
+    minWidth: 120
+  },
+  group: {
+    margin: `${theme.spacing.unit}px 0`
+  }
+});
+
+const {
+  sections,
+  painConditionData,
+  medicationData,
+  efficacyLevel,
+  proceduresData,
+  nonPharmaData
+} = painHistoryData;
+
+const EditDialog = props => {
+  const { classes, parentState, updateParentState } = props;
+
+  return (
+    <div>
+      <FormControl required className={classes.formControl}>
+        <FormLabel component="legend">{sections.painCondition.title}</FormLabel>
+        <Select
+          value={parentState.painCondition}
+          onChange={updateParentState}
+          name="painCondition"
+          displayEmpty
+          inputProps={{
+            id: "painCondition-required"
+          }}
+          className={classes.selectEmpty}
+        >
+          {painConditionData.map(condition => (
+            <MenuItem key={condition.value} value={condition.label}>
+              {condition.label}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <FormControl required className={classes.formControl}>
+        <FormLabel component="legend">
+          {sections.medication.whatHaveTried}
+        </FormLabel>
+        <Select
+          value={parentState.medicationName}
+          name="medicationName"
+          onChange={updateParentState}
+          displayEmpty
+          className={classes.selectEmpty}
+        >
+          {medicationData.map(medication => (
+            <MenuItem key={medication.value} value={medication.label}>
+              {medication.label}
+            </MenuItem>
+          ))}
+        </Select>
+      </FormControl>
+      <FormControl required className={classes.formControl}>
+        <FormLabel component="legend">
+          How did medication {parentState.medicationName} worked?
```

Comment:
worked => work
```
How did medication {parentState.medicationName} work?
```

Reply:
We can avoid the word medication, because the name of the medication will be there. What about this: 

How did the {parentState.medicationName} work?

An example of what the user will see is: "How did the Ibuprofen work?"